chord_metadata_service/discovery/fields.py

```diff
@@ -2,47 +2,21 @@
 
 from calendar import month_abbr
 from collections import Counter, defaultdict
-from django.db.models import Case, CharField, Count, F, Func, IntegerField, Model, QuerySet, When, Value
+from django.db.models import Case, CharField, Count, F, Func, IntegerField, QuerySet, When, Value, Q
 from django.db.models.functions import Cast
-from typing import Any, Mapping, Type
+from typing import Any, Mapping
 
 from ..logger import logger
 
 from . import fields_utils as f_utils
 from .censorship import get_threshold, thresholded_count
 from .fields_utils import monthly_generator
```

Comment:
I know it's unrelated to this PR somewhat, but can you replace the `monthly_generator` use with an `f_utils.monthly_generator` and remove this line? it's bothering me looking at the diff below... 

otherwise, lgtm!